lib/stripe/api_resource.rb

```diff
@@ -48,6 +48,30 @@ def self.save_nested_resource(name)
       end
     end
 
+    # Adds a custom method to a resource class. This is used to add support for
+    # non-CRUDL API requests, e.g. capturing charges. custom_method takes the
```

Comment:
Oh man, thought this was a typo, but TIL about CRUDL.

Reply:
Haha, I wasn't even sure that was really a thing TBH :)